examples/python/remote_storage_example/README.md

```diff
@@ -0,0 +1,110 @@
+# NIXL Storage Transfer Sample
+
+A high-performance storage transfer system built on NIXL (NVIDIA Inference Xfer Library) that demonstrates local and remote storage operations using POSIX and GDS (GPU Direct Storage) backends.
+
+## Features
+
+- **Flexible Storage Backends**
+  - GDS (GPU Direct Storage) support for high-performance transfers
+  - POSIX fallback for standard storage operations
+  - Automatic backend selection based on availability
+
+- **Transfer Modes**
+  - Local memory-to-storage transfers
+  - Remote memory-to-storage transfers
+  - Bidirectional operations (READ/WRITE)
+  - Batch processing support
+
+- **Network Communication**
+  - UCX-based data transfer
+  - Metadata exchange between nodes
+  - Asynchronous notification system
+
+## Project Structure
+
+- `nixl_storage_utils/` - Core module providing storage utilities
+  - `__init__.py` - Module initialization and exports
+  - `common.py` - Common utilities and shared functionality
+- `nixl_p2p_storage_example.py` - Main application that can run as either initiator or target
+
+## Requirements
+
+- Python 3.6+
+- NIXL library with the following plugins:
+  - GDS (optional)
+  - POSIX
+  - UCX
+
+## Usage
+
+The system operates in two modes: client and server.
+
+The servers wait for requests from clients so it can READ/WRITE from its storage to a remote node.
+
+The initiator initiates transfers and can perform both local and remote operations with storage servers.
+
+### Running as Client
+
```

Comment:
Add a line about how to install packages like gds installation/pip install nixl and provide links here